rancher/concepts/health-checks/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Health Checks with Rancher
+layout: rancher-default
+
+---
+
+#### Health Checks
+
+Rancher implements a distributed health monitoring system by running an HAProxy instance on every host for the sole purpose of providing health checks to containers.  When health checks are enabled either on an individual container or a service, each container is then monitored by up to three HAProxy instances running on hosts seperate to that containers parent host. The container is considered healthy if at least one HAProxy instance reports a "passed" health check.
```

Comment:
We should clarify that haproxy instance is a really Rancher network agent with haproxy software installed. Otherwise users might expect designated container just for healtcheck service purpose.